src/BackupExplorer/Parser.Tests/StateManagerTests.cs

```diff
@@ -189,6 +198,9 @@ public async Task BackupParser_StateManagerAbleToReadQueuesInTransactionApplied(
         {
             using (var backupParser = new BackupParser(BackupFolderPath, ""))
             {
+                {
+
+                };
```

Comment:
What is this for?

Reply:
Will remove these. Its missing code cleanup